TLM/.editorconfig

```diff
@@ -2,11 +2,9 @@ root=true
 
 [*.cs]
 max_line_length=100
-tab_width=8
 insert_final_newline=true
 trim_trailing_whitespace=true
 indent_style=space
-indent_size=4
```

Comment:
If this file is intentional, the `indent_size = 4` should be retained unless you add `.cs` to the big wildcard clause around line 99.

Reply:
```
[*.cs]
max_line_length=100
insert_final_newline=true
trim_trailing_whitespace=true
indent_style=space

[*]
charset=utf-8
end_of_line=crlf
indent_style=space
indent_size=4  <----- LOOK HERE
```
[*] wild card is already taking care of it. Don't worry I tried it and it works :)